src/structures/Guild.js

```diff
@@ -177,6 +177,8 @@ class Guild extends Base {
      * * VERIFIED
      * * VIP_REGIONS
      * * WELCOME_SCREEN_ENABLED
+     * * MEMBER_VERIFICATION_GATE_ENABLED
+     * * PREVIEW_ENABLED
```

Comment:
Shouldn't these be sorted alphabetically just like the rest was? Same goes for the typings file.

Reply:
Guess I put them in the order discord docs have them in without noticing they're alphabetical here. Fixed!